### ISSUE TITLE
diaspora* / mastodon workshop

### ISSUE THREAD

**Issue by cappert** [lyonrb/wishlist#61]
after the JDLL workshop we talked about doing a workshop based around Diaspora* as the bug triage and documentation is of ☼masterful quality☼.
mastodon is another decentralised social network and it's all the hype these days so I'm bundling it here.
cc @lcguida @Flaburgan

**Comment by Flaburgan** [comment]
Yep, I can definitely introduce diaspora* to anyone who wants, even if I'm a front-end dev, not a Rails one ;)

**Comment by lcguida** [comment]
And after that we could go further and make a Open Source night, to get together and solve some issues maybe.

Could on me for help if needed.
